setup.py

```diff
@@ -23,16 +23,17 @@
     command_number = 'git rev-list --count HEAD'
 
     try:
-        commit_hash = subprocess.check_output(command, shell=True).decode('ascii').strip()
-        commit_number = subprocess.check_output(command, shell=True).decode('ascii').strip()
+        commit_hash = subprocess.check_output(command_hash, shell=True)\
+            .decode('ascii').strip()
```

Comment:
I this this is a case where breaking the line doesn't help readability - could you remove the line break? (also in the line below)